Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -309,7 +311,36 @@ public async Task CloseIfNeededAsync(SessionInfo session)
                 _activeConnections.TryRemove(key, out _);
                 if (!string.IsNullOrEmpty(session.PlayState?.LiveStreamId))
                 {
-                    await _mediaSourceManager.CloseLiveStream(session.PlayState.LiveStreamId).ConfigureAwait(false);
+                    bool liveStreamNeedsToBeClosed = false;
```

Comment:
Now that this is more then 1 line please extract it (and L1046-L1078) to a common function